jhipster-framework/src/test/java/tech/jhipster/config/h2/H2ConfigurationHelperTest.java

```diff
@@ -0,0 +1,24 @@
+package tech.jhipster.config.h2;
+
+import org.junit.jupiter.api.Test;
+
+import java.net.URL;
+import java.net.URLConnection;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class H2ConfigurationHelperTest {
+
+    @Test
+    void initH2Console() {
```

Comment:
We could add the assertThatCode, which doesn't throw exception, so Sonar will be happy

Reply:
Similar to https://github.com/jhipster/generator-jhipster/pull/13680